packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -450,7 +456,7 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 				this.reSubmit(content, localOpMetadata);
 			},
 			applyStashedOp: (content: any): unknown => {
-				return this.applyStashedOp(content);
+				return this.applyStashedOp(parseHandles(content, this.serializer));
```

Comment:
Same as above - I don't think this belongs in this PR.

Reply:
the goal is the op comes back is the same way it left. since it is idempotent this doesn't hurt, and it would be best if dds didn't need to worry about it at all, its weird to say they need to decode but shouldn't encode